### ISSUE TITLE
FR: Capitalize book title and current chapter in bottom status bar

### ISSUE THREAD

**Issue by debashisbiswas** [koreader/koreader#9345]
**Does your feature request involve difficulty completing a task? Please describe.**
When displaying the book title and the current chapter in the status bar, the user does not have the option to display the text with all uppercase letters.

**Describe the solution you'd like**
When the user goes into to `Settings > Status bar > Settings > Font`, they will see a checkbox that allows them to toggle uppercase text in the bottom status bar.

**Describe alternatives you've considered**
It seems that there is also have the option of applying patches as of #9104, which is available as an alternative.

**Additional context**
I've pulled the source code and uppercased the string directly, but haven't added a setting for it.  I can move further with this change if you think it's worth adding to the project.
Here is a screenshot of what I am describing:

![statusbar_capital](https://user-images.githubusercontent.com/37234687/179435054-0d15bf69-bf2f-4ba2-a0bc-25f8fab10f9d.png)
 

Something to discuss is whether this should also affect the "Custom text" option on the status bar.

**Comment by Frenzie** [comment]
Sidenote, but it'll probably look better with small caps.
<img src=https://user-images.githubusercontent.com/202757/179449580-2c80499b-f9d1-4c2e-9fc8-bee7488e9c5b.png width=45%>

Screenshot taken like this but of course this way that applies anywhere xgettext is used.
```
diff --git a/frontend/ui/font.lua b/frontend/ui/font.lua
index 67514cf25..35572e5f1 100644
--- a/frontend/ui/font.lua
+++ b/frontend/ui/font.lua
@@ -321,7 +321,7 @@ function Font:getFace(font, size, faceindex)
         -- them in the UI.)
         -- Use HB defaults, and be sure to enable kerning and ligatures
         -- (which might be part of HB defaults, or not, not sure).
-        face_obj.hb_features = { "+kern", "+liga" }
+        face_obj.hb_features = { "+kern", "+liga", "+smcp" }
         -- If we'd wanted to disable all features that might be enabled
         -- by HarfBuzz (see harfbuzz/src/hb-ot-shape.cc, quite unclear
         -- what's enabled or not by default):
```

**Comment by melyux** [comment]
I just had this thought while reading. I was like, why doesn't this footer look as nice as Kobo's native one? It's the capitalization, dummy. I like the small caps idea too.

**Comment by debashisbiswas** [comment]
Right, I had this thought after switching over from the Kobo's native reader as well. Here's an example of what it looks like on there.

![image](https://user-images.githubusercontent.com/37234687/180663742-d81d069b-5275-4a9d-a873-42215b893d92.png)

I'd prefer regular caps over the small caps, but that's just my preference. Maybe I'm biased after getting used to the native Kobo design
